staking-cli/src/demo.rs

```diff
@@ -78,8 +106,12 @@ pub async fn stake_for_demo(config: &Config) -> Result<()> {
         .await?;
         assert!(receipt.status());
 
-        let receipt = delegate(stake_table, validator_address, amount).await?;
+        tracing::info!(
+            "delegate {delegate_amount_esp} ESP for validator {val_index} from {validator_address}"
+        );
+        let receipt = delegate(stake_table, validator_address, delegate_amount).await?;
```

Comment:
so is this delegating to itself? can we also have multiple delegators like some random ethereum wallets delegating their stake to each validator.